openc3/lib/openc3/models/scope_model.rb

```diff
@@ -252,6 +276,21 @@ def deploy_scopecleanup_microservice(gem_path, variables, parent)
       Logger.info "Configured microservice #{microservice_name}"
     end
 
+    def deploy_critical_cmd_microservice(gem_path, variables, parent)
+      microservice_name = "#{@scope}__CRITICALCMD__#{@scope}"
```

Comment:
Need a corresponding `model.destroy` in `undeploy`

Reply:
Good catch.